shared/events.js

```diff
@@ -280,6 +281,13 @@ export class GroupContract extends HashableContract {
       },
       AcknowledgeFounder (state, data) {
         if (!state.members.find(username => username === data.username)) { state.members.push(data.username) }
+      },
+      // TODO: remove group profile when leave group is implemnted
+      ProfileAdjustment (state, data) {
+        let profile = state.profiles[data.username]
+        if (!profile) { profile = state.profiles[data.username] = {} }
+        if (!data.value) { return delete profile[data.name] }
+        profile[data.name] = data.value
```

Comment:
Sorry I thought I got rid of this. I initially made this fix in my profile branch